### PR TITLE
checkup: Increase Setup timeout

### PR DESCRIPTION
**Pull request by RamLavi**
The checkup sometimes fails on setup timeout, although nothing seems to be wrong.
Increasing the timeout to 15m.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -36,7 +36,7 @@ build:
 	           --workdir $(CURDIR) \
 	           -e GOOS=linux \
 	           -e GOARCH=amd64 \
-	           $(GO_IMAGE_NAME):$(GO_IMAGE_TAG) go build -v -o $(BIN_DIR)/$(CHECKUP_IMAGE_NAME) ./cmd/
+	           $(GO_IMAGE_NAME):$(GO_IMAGE_TAG) go build -buildvcs=false -v -o $(BIN_DIR)/$(CHECKUP_IMAGE_NAME) ./cmd/
```
Comment (orelmisan) [inline Makefile:39]: This change doesn't seem to be related to the PR.

File: tests/checkup_test.go [hunk 2]
```diff
@@ -42,7 +42,7 @@ const (
 	testConfigMapName                   = "dpdk-checkup-config"
 	testCheckupJobName                  = "dpdk-checkup"
 
-	testTimeout = 10 * time.Minute
+	testTimeout = 11 * time.Hour
```
Comment (orelmisan) [inline tests/checkup_test.go:45]: What is the reason to move from 10 minutes to eleven hours?

File: tests/checkup_test.go [hunk 3]
```diff
@@ -291,7 +291,7 @@ func newConfigMap() *corev1.ConfigMap {
 		"spec.timeout": testTimeout.String(),
 		"spec.param.networkAttachmentDefinitionName": networkAttachmentDefinitionName,
 		"spec.param.trafficGenPacketsPerSecond":      "8m",
-		"spec.param.testDuration":                    "1m",
+		"spec.param.testDuration":                    "4h",
```
Comment (orelmisan) [inline tests/checkup_test.go:294]: This change doesn't seem to be related to the PR.

**Review by orelmisan — CHANGES_REQUESTED** [review]
Thank you for the PR @RamLavi 
Please see the inline comments.

There is a typo in the commit message and PR description.

**Comment by RamLavi** [comment]
ahh sorry I added to many changes I did not intend.. will fix soon and ping you

File: tests/checkup_test.go (hunk 3 above)
Comment (RamLavi) [reply tests/checkup_test.go]: DONE

File: tests/checkup_test.go (hunk 2 above)
Comment (RamLavi) [reply tests/checkup_test.go]: don't not need to be there. DONE

File: Makefile (hunk 1 above)
Comment (RamLavi) [reply Makefile]: nope

**Comment by orelmisan** [comment]
> Thank you for the PR @RamLavi Please see the inline comments.
> 
> There is a typo in the commit message and PR description.

`Yhe checkup` -> `The checkup`

**Comment by RamLavi** [comment]
[Change](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/8f06444c94b858e520d0faf9ff7621f287d8ac41..39b6261d3680c5cb17ac6d44ef63d11dea9d674b): Fix commit typo

**Review by orelmisan — APPROVED** [review]

**Comment by RamLavi** [comment]
Passed CI on OCP 4.14 cluster with modified kubevirt yo override SMT alignment error:
```
make test/e2e
podman run --rm \
           --volume /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:Z \
           --volume /home/ralavi/.kube:/root/.kube:Z,ro \
           --workdir /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup \
           -e KUBECONFIG=/root/.kube/kubeconfig \
           -e TEST_CHECKUP_IMAGE=quay.io/ramlavi/kubevirt-dpdk-checkup:latest \
           -e TEST_NAMESPACE=dpdk-checkup-ns-1 \
           -e NETWORK_ATTACHMENT_DEFINITION_NAME=dpdk-sriovnetwork-ns-1 \
           -e TRAFFIC_GEN_CONTAINER_DISK_IMAGE=quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:main \
           -e VM_UNDER_TEST_CONTAINER_DISK_IMAGE=quay.io/kiagnose/kubevirt-dpdk-checkup-vm:main \
           docker.io/library/golang:1.20.6 go test ./tests/... -test.v -test.timeout=1h -ginkgo.v -ginkgo.timeout=1h
=== RUN   TestKubevirtDpdkCheckup
Running Suite: KubevirtDpdkCheckup Suite - /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests
==============================================================================================================
Random Seed: 1700392264

Will run 1 of 1 specs
------------------------------
[BeforeSuite] 
/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests/test_suite_test.go:56
[BeforeSuite] PASSED [0.002 seconds]
------------------------------
Execute the checkup Job should complete successfully
/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests/checkup_test.go:83
• [527.898 seconds]
------------------------------

Ran 1 of 1 Specs in 527.900 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestKubevirtDpdkCheckup (527.90s)
PASS
ok  	github.com/kiagnose/kubevirt-dpdk-checkup/tests	527.925s
```
